views/partners.handlebars

```diff
@@ -0,0 +1,117 @@
+      <img src="images/logos-CA/COVID-19 Resources Canada.svg" alt="COVID-19 Resources Canada" width="140">
+    </a>
+    <a href="https://www.donnezlaprotection.com/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Donnez La Protection.png" alt="Donnez La Protection" width="140">
+    </a>
+    <a href="https://jkwanmd.com/ppe" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Dr. Jennifer Kwan.png" alt="Dr. Jennifer Kwan" width="140">
+    </a>
+    <a href="https://maskaherobc.ca/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Mask A Hero BC.png" alt="Mask A Hero BC" width="140">
+    </a>
+    <a href="https://maskaheroontario.ca/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Mask a Hero Ontario.png" alt="Mask a Hero Ontario" width="140">
+    </a>
+    <a href="https://www.mask-match.com/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Mask Match.png" alt="Mask Match" width="140">
+    </a>
+    <a href="http://frontlinemasks.ca/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Mobilizing Masks for Health Care Providers.png" alt="Mobilizing Masks for Health Care Providers" width="140">
+    </a>
+    <a href="https://www.nosm.ca/our-community/nosm-student-council/nosmsc-covid-19-relief/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Northern Ontario PPEforHCP.png" alt="Northern Ontario #PPEforHCP" width="140">
+    </a>
+    <a href="https://ppeforhcpsto.ca/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/PPE for HCPs Toronto Team.png" alt="PPE for HCPs Toronto Team" width="140">
+    </a>
+    <a href="https://www.redmooncosplaysolutions.com/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Red Moon Cosplay.jpg" alt="Red Moon Cosplay" width="140">
+    </a>
+    <a href="https://shield-net.org/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Shield Net.png" alt="Shield Net" width="140">
+    </a>
+    <a href="http://thehomefront.ca/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/The Home Front.jpg" alt="The Home Front" width="140">
+    </a>
+    <a href="http://www.trackppe.com/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Track PPE.png" alt="Track PPE" width="140">
+    </a>
+    <a href="www.theppedrive.com" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/thePPEDrive.png" alt="thePPEDrive" width="140">
+    </d>
+    <a href="www.fb.com/groups/VolunteersinOntarioFindingPPEforHospitals/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Volunteers in Ontario Finding PPE  for Hospitals.jpg" alt="Volunteers in Ontario Finding PPE  for Hospitals.jpg" width="140">
```

Comment:
you can do i18n for alt attrs as well. `class="i18n" data-i18n="[alt]ftm-my-key"` should work

Reply:
Since most of these are names of organizations, does it make sense to translate them?